.travis/build-site.sh

```diff
@@ -13,21 +13,9 @@ fi
 
 
 (
-    export PING_SLEEP=30s
-    export BUILD_OUTPUT=/tmp/build-site.out
-    export PING_PID_FILE=/tmp/build-site-ping.pid
-
-    source .travis/background-job-funcs.sh
-
     # Run the build, redirect output into the file
-    ./mvnw install -DskipTests=true -B -V >> $BUILD_OUTPUT 2>&1
-    ./mvnw site site:stage -Psite -B -V >> $BUILD_OUTPUT 2>&1
-
-    # The build finished without returning an error so dump a tail of the output
-    dump_output
-
-    # nicely terminate the ping output loop
-    kill_ping
+    travis_wait ./mvnw install -DskipTests=true -B -V
```

Comment:
Reading the code of travis_wait: Ok, the initial problem (too much logging) is solved - it's also written into a temporary file. However, they [cat the logfile](https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/templates/header.sh#L203) - so, we still might run into the log size limitation...